index.mustache

```diff
@@ -115,14 +115,12 @@
       <nav class="footer-navigation">
         <i class="icon-square-footer"></i>
         <ul class="footer-navigation-list">
-          <li class="footer-navigation-item"><a href="https://squareup.com/directory">Square Directory</a></li>
-          <li class="footer-navigation-item"><a href="https://cash.me">Square Cash</a></li>
-          <li class="footer-navigation-item"><a href="https://squareup.com/register">Square Register</a></li>
```

Comment:
Did you mean to remove all these links too?

Reply:
Cash has their own GitHub, and `/register` redirects to `/point-of-sale`. 

I guess we could add in a link to the Cash GitHub in there if we really wanted.

Reply:
Added a link to the Cash Github.

Reply:
Cash has their own github, but no reason to not point people there from this page.